app/views/admin/enterprises/form/_users.html.haml

```diff
@@ -45,13 +45,30 @@
           %td
             = hidden_field_tag "enterprise[user_ids][]", nil, multiple: true, 'ng-value' => 'manager.id'
             {{ manager.email }}
-            %i.confirmation.confirmed.fa.fa-check-circle{ 'ofn-with-tip' => t('.email_confirmed'), ng: {show: 'manager.confirmed'} }
+            = render partial: 'admin/shared/special_tooltip', locals: {tooltip_text: t('.email_confirmed'), icon_class: 'confirmation confirmed fa fa-check-circle'}
+            /%i.confirmation.confirmed.fa.fa-check-circle{ 'ofn-with-tip' => t('.email_confirmed'), ng: {show: 'manager.confirmed'} }
```

Comment:
It looks like this line was commented out but should be removed.

Reply:
Actually, is this whole table intended to be removed? It seems to be built by angular, and the new unordered list below seems to be the replacement.

Reply:
Actually, I totally miss this one. As was focused on the modal. 
Will deep dive into. 

Not so simple to tackle someone stuff!